plugins/inputs/snmp/snmp.go

```diff
@@ -25,8 +25,13 @@ import (
 const description = `Retrieves SNMP values from remote agents`
 const sampleConfig = `
   ## Agent addresses to retrieve values from.
+  ##   format:  agents = ["<scheme://><hostname>:<port>"]
+  ##   scheme:  optional, either udp, udp4, udp6, tcp, tcp4, tcp6.            
+  ##            default is udp
+  ##   port:    optional
   ##   example: agents = ["udp://127.0.0.1:161"]
   ##            agents = ["tcp://127.0.0.1:161"]
+  ##            agents = ["udpv4://v4only-snmp-agent"]
```

Comment:
extra v here too

Reply:
Uhhh. 
How many Typos can there be in a few-line-change! 
Thanks for your input! :)

Reply:
That's what reviews are for!

Hey I noticed you said you're just getting started with go.  If you're looking for more opportunities to write go code, feel free to grab issues with the "enhancement" or "good first issue" label.  The influx and community devs and I are happy to give you feedback on PRs.

Thanks again for this PR!